cluster/src/main/java/org/apache/iotdb/cluster/rpc/raft/impl/RaftNodeAsClientManager.java

```diff
@@ -207,7 +266,7 @@ public Executor getExecutor() {
       } catch (RemotingException | InterruptedException e) {
         LOGGER.error(e.getMessage());
         qpTask.setTaskState(TaskState.EXCEPTION);
-        releaseClient();
+        releaseClient(this);
```

Comment:
why not RaftNodeAsClient.this?

Reply:
It's the same, but pointing it out clearly is better.